presto-hive/src/test/java/com/facebook/presto/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -208,7 +208,7 @@ public void testNoHangIfPartitionIsOffline()
                 .collect(toImmutableList());
     }
 
-    private static List<HiveSplit> drainSplits(HiveSplitSource source)
+    static List<HiveSplit> drainSplits(HiveSplitSource source)
```

Comment:
Usually we just put tests in `TestBackgroundHiveSplitLoaderForBucketedTables.java` into this `TestBackgroundHiveSplitLoader.java` class.

In this specific case, I understand the tests and other helper methods in `TestBackgroundHiveSplitLoaderForBucketedTables.java` are very specific and may not suit for general purposed `TestBackgroundHiveSplitLoader.java`. But a preferred approach might just be refactoring these methods into a helper class like `HvieSplitTestHelper` (this name is just made up)

Reply:
Makes sense - I will move it to a separate PR